README.md

```diff
@@ -58,68 +24,58 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
+# 📖 Crpyto-Cash is a web application and a platform that utilizes the Coingecko API to fetch data on different cryptocurrencies and displays the information for each coin in a comprehensive manner.</a>
 
-**[your_project__name]** is a...
 
 ## 🛠 Built With <a name="built-with"></a>
+- React
+- API
+- Webpack
+- NPM
+- Redux
+- Jest
 
 ### Tech Stack <a name="tech-stack"></a>
 
-> Describe the tech stack and include only the relevant sections that apply to your project.
-
 <details>
   <summary>Client</summary>
   <ul>
-    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a>HTML</a></li>
+    <li><a>CSS</a></li>
+    <li><a>React</a></li>
   </ul>
 </details>
 
 <details>
   <summary>Server</summary>
   <ul>
-    <li><a href="https://expressjs.com/">Express.js</a></li>
+    <li><a>Crypto Cash</a></li>
   </ul>
 </details>
 
-<details>
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- Shows list of all available coins listed
+- Shows current price and percentage change in 24hr for each coin in its homepage
+- Show more details of each coin when clicked
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://yourdeployedapplicationlink.com)
+- [Live Demo Link](https://crypto-cash-website.netlify.app/)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- GETTING STARTED -->
+## 🚀 Video Description <a name=""></a>
+- [Video](https://youfiles.herokuapp.com/videodictionary/?m=Video_Player_Drive)
+
 
```

Comment:
- [ ] Dear @lawrahkonwea, Your video documentation is not available from this link. Please make sure to record a video about the project having `maximum length of the video is 5`, `contains a clear and objective description`, `contains a demo of the project,` and `contains a highlight about a piece of the code you finds interesting`